Signal/src/ViewControllers/NewGroupViewController.m

```diff
@@ -102,6 +102,8 @@ - (void)loadView
 
     self.title = [MessageStrings newGroupDefaultTitle];
 
+    self.view.backgroundColor = UIColor.whiteColor;
```

Comment:
I suspect this should be:

```
    self.view.backgroundColor = Theme.backgroundColor;
```
Maybe I'll move that into `OWSViewController` so we can dry this up though.

Reply:
Oh sorry, this is for v2.28. Forget it.

Reply:
You're right - updated.